### ISSUE TITLE
Plugin fails to build after flatlaf migration

### ISSUE THREAD

**Issue by YvesW** [Loze-Put/hunllef-helper#43]
hunllef-helper will soon fail to build; the new laf was temporarily released today (and rolled back) and a lot of people complained about missing this plugin.

You can probably use ``onActivate`` and ``onDeactivate`` instead of ``navigationButton.isSelected()``, something like:
```java
	@Override
	public void onActivate()
	{
		super.onActivate();
		boolean = true;
	}
```

Instead of
```java
				SwingUtilities.invokeLater(() ->
				{
					if (!navigationButton.isSelected())
					{
						navigationButton.getOnSelect().run();
					}
				});
```

you can probably use:

```java
				SwingUtilities.invokeLater(() -> clientToolbar.openPanel(navButton));
```

Currently you can set your version to ``'1.10.20-SNAPSHOT'`` (edit: ``latest.release`` also works now) to check out the upcoming flatlaf changes.

**Comment by Loze-Put** [comment]
Thanks for the heads up. I'll take a look at it and I'll create an update for the plugin.

**Comment by Loze-Put** [comment]
Fixed in #45
